### DISCUSSION TITLE
如果人有超能力

### DISCUSSION THREAD

**Discussion by zhilin93517** [1101-GS4719-C/Python-Community-GS4719-C#2156] [General]
如果每個人都可以自己選擇一個超能力的話，你們最想要有甚麼超能力?

超能力的定義是指一個人擁有特殊或非凡的超人能力，其能力大於正常水平，且常人不擁有的能力。

如果是我的話我最想要擁有飛行的能力，因為我不喜歡花時間走路或騎腳踏車，我希望去哪裡都可以用非常快的速度到達，並且幾乎不須出力。因此我認為飛行是非常適合我的一個超能力。

**Comment by gigitilto** [comment]
我希望我可以擁有替身，替身不會被一般人看到，可以讓替身做很多事，替身就相當於是我的分身，替身受傷我也會受傷，但替身會有很強的能耐，例如力氣很大、速度很快，甚至於暫停時間，一旦有了替身就可以做到很多事，讓我無所不能，所以這個超能力我覺得很適合我。

**Comment by Mminniehuang** [comment]
說到超能力，我一直以來的答案都是瞬間移動。
有瞬間移動的話就可以在上班上學的時候避開塞車、擁擠的人潮，節省各種通勤時間來做更多想做的事，也可以想去哪就去哪，任何時候都可以來一段說走就走的旅行，還可以不用帶行李，需要什麼隨時都可以回家拿再回來！
我沒有想過飛行這個超能力，現在想想可能是因為稍微有點怕高，不過飛去目的地雖然可能沒有瞬間移動那麼快，但感覺比較刺激也比較浪漫🤣

**Comment by wcyeee** [comment]
如果要有超能力的話我可能會想要有讀心術！
感覺對於現在的社會，溝通還是最重要的部分，如果我可以輕易瞭解他人的想法，感覺我會可以很輕易的達成任何事物。（例如某個有錢人的銀行密碼或是某個商人到底想要什麼交易條件、目的是什麼）（或是另一半到底在想什麼） 並且如果我的溝通能力也很好的話，配合讀心術感覺就是一個催眠大師，很方便也對我的生活很有益。（只是也許有些時候會失去刺激感，畢竟別人在想什麼我很輕易就知道了）

**Comment by Jinyuu17** [comment]
如果我能擁有一種超能力，我想要「即時理解和學習一切知識」的能力。這樣我能夠瞬間掌握任何領域的知識、語言、技術，並應用它來解決各種問題，幫助別人，推動科學和技術的進步。成為真人版的聊天機器人。

**Comment by CeBru13** [comment]
如果我能擁有一種超能力，我想要有預知未來的能力，但這個未來是可以透過我的選擇而改變，比較像是奇異博士那樣可以看見未來並進行選擇，可以讓我做出最棒的決定

**Comment by 112121py** [comment]
如果可以選一個超能力，我應該會想選類似哆啦A夢的任意門那種，可以在A地開一道門就能抵達B地，對於大學生來說，不管是上課快遲到還是住的離家裡很遠的人都蠻有幫助的，可以節省很多開銷，不過當然也有蠻多問題的啦xd

**Comment by seastar0687** [comment]
「如果我有超能力」這個假設問題是一個歷久不衰的經典題目，我認為其原因是因為人總是對於日常生活有著大大小小的不滿，而對於思考解決這些不滿最簡單的方法就是設想有一個超越人智的力量來解決，這種思考能快速代謝情緒，某部分來說是十分有利的思考，但如果過度沈醉其中就會變得逃避現實。
順帶一提，我自己會想要變得更聰明，畢竟這個世界上大部分的事物都是可以依靠足夠進步的科技達成

**Comment by DarsooK** [comment]
能過產生無窮無盡的金錢，現在的社會做任何事都需要金錢：飲食、交通、就學……，人生許多時候為金錢所迫，無法追尋、探索自己真正理想的模樣，而有無窮無盡的金錢或許就可以尋找自我和這世界的真理吧。

**Comment by forestdew507** [comment]
我也想要有預知未來的能力 ! 當對一些事情過度糾結，就可以用這個超能力讓過不去的自己在更短的時間內解脫，這能力也能讓金錢不再成為困擾，不過也不是處處都拿來使用，畢竟有時候未知本身好像也滿有吸引力的XD

**Comment by 113607514** [comment]
瞬間移動是我的夢幻的超能力，能讓人瞬間出現在世界的任何角落，免去旅途的煩惱和等待。擁有這種能力，不僅能快速完成日常行程，還能隨心所欲探索新地方、造訪遠方的朋友，真正實現「心之所至、身亦能至」的自由。

**Comment by lai1008** [comment]
瞬間移動是我的夢幻的超能力，可以想要去哪裡就到哪裡，還可以不用考慮時間，這樣對要返鄉的大學生就更加的方便，也省下許多的時間。

**Comment by yuan3x2** [comment]
我希望有最強大腦，感覺可以少痛苦一點，讀書背書也會比較能靈活運用，還可以節省補習費以及賺取獎學金，可不可以給我一顆最強大腦(來自托福苦守的淚目。

**Comment by Boying12** [comment]
我最想擁有的就是瞬間移動的能力。想像一下，無論是工作、學習還是旅行，瞬間移動都能讓我在眨眼之間到達任何地方，省去長時間的交通和等待。我可以隨時飛往心儀的城市、欣賞美麗的風景，或是與朋友們在不同的地方聚會，獲得更多的生活體驗。

**Comment by yue083139** [comment]
我應該會想擁有分身的能力吧，上大學之後不像高中一樣只要專注在學習上就好，還有人際壓力及活動等等的事情需要兼顧且同時不能落下越來越困難的學業，有時候會真心覺得如果有人能幫我做的話就太好了，或許能夠讓我有多一些的喘息時間吧。

**Comment by Elaine4488** [comment]
我會選擇穿越時空的能力。這樣我就可以回到過去，修正一些錯誤或是跟過去的人重逢，甚至可以未來看一下，了解世界會發展成什麼樣子。除了可以探索歷史，也能讓我提前準備，面對即將來臨的挑戰。

**Comment by 113607514** [comment]
如果能有一個超能力，我想都不用想，肯定是**瞬間移動**。光是想到可以省下每天擠公車、趕捷運的時間，就覺得生活幸福感會瞬間提升不少。下課一結束，「唰」地一下就到家，再也不用擔心晚睡早八遲到這種悲劇（笑）。  

而且旅行會變得超方便！想吃東京的拉麵？下一秒人已經坐在新宿的小店裡了。想看歐洲的雪景？刷個牙的時間就能飛到阿爾卑斯山腳下。這種自由自在的感覺，光想就很爽快。  

不過仔細想想，擁有超能力感覺也會讓生活少了點挑戰。以前去旅行，光是攢錢、規劃路線就讓人期待半天。如果一瞬間就能到達，好像少了些過程中的樂趣。但說真的，如果真的能瞬移，我還是會毫不猶豫地接受這個超能力，畢竟早上多睡十分鐘，才是學生時期最需要的超能力之一吧！

**Comment by Taoyi-Chiang** [comment]
我這個人比較沒這麼貪心，我只要獲得操控股市的鈔能力就可以了wwwwww

**Comment by poyen0806** [comment]
各位或許知道，在各個國家存在著雙休日或是三休日。
在來之不易的休息日裡，人們得以從生活的重壓中暫時解脫。
也只有在休息日裡，人們不必面對社會中的弱肉強食，能夠在這短短的數日中幸福的活著。
但在我看來，完美的社會制度應該是七休日。
在星期日的明天，是第二、第三、乃至永遠的又一個星期日。

所以如果我能選一個超能力，我想創造一個這樣的世界，充滿假日，我假日太少了...
